redis-persistence/src/main/java/com/netflix/conductor/dao/dynomite/RedisMetadataDAO.java

```diff
@@ -80,10 +80,6 @@ public String updateTaskDef(TaskDef taskDef) {
     }
 
     private String insertOrUpdateTaskDef(TaskDef taskDef) {
```

Comment:
Is this API being used from other resource then Metadataservice? If, yes then you might want to keep the code.

Reply:
it's not